testData/solutions/win32.json5

```diff
@@ -3206,6 +3206,33 @@
         }
     },
 
+    "net.gpii.psp": {
+        "name": "Personal Settings Panel",
+        "contexts": {
+            "OS": [{
+                    "id": "win32",
+                    "version": ">=5.0"
+            }]
+        },
+        "settingsHandlers": {
+            "configuration": {
+                "type": "gpii.settingsHandlers.webSockets",
+                "liveness": "live",
+                "options": {
+                    "path": "net.gpii.psp"
+                },
+                "supportedSettings": {
+                    "language": {},
```

Comment:
Don't we need to include the metadata for this settitng? I guess that we want it to be there if we want to get it rendered in the PSP.

Reply:
Also in order to support editing it in the PPT